### PR TITLE
Features/1040 old panamenian emails

### PR DESCRIPTION
**Pull request by smpiano**
Read the `README.md` to understand what does the command.
This must not be deployed to `GAE standard environment`, it is planned to run in local servers.

Related with> https://github.com/GlobalFishingWatch/GFW-Tasks/issues/1040

### CODE REVIEW

File: imap/read.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            print '>> ', att_name
+
+            #Upload attachment to GCS
+            transfer = GCSTransfer(to, msg_date.strftime("%Y-%m-%d"))
+            path = transfer.local_transfer(att_name, content)
+
+
+def main(account, folder, start_date, end_date):
+    print 'Connects to GMAIL imap'
+    M = imaplib.IMAP4_SSL('imap.gmail.com')
```
Comment (enriquetuya) [inline imap/read.py]: @smpiano **NEVER** call a variable with a letter, it should be a word that has some meaning, if not the code is really hard to follow.  Change here and in `process_mailbox` method.

Comment (smpiano) [reply imap/read.py]: Ok, completly agreed. I followed the suggestions from the source https://docs.python.org/2/library/imaplib.html
